dcicutils/sheet_utils.py

```diff
@@ -128,24 +157,128 @@ def set_path_value(cls, datum: Union[List, Dict], path: ParsedHeader, value: Any
             cls.set_path_value(datum[key], more_path, value)
 
 
-class WorkbookManager:
+class AbstractTableSetManager:
+    """
+    The TableSetManager is the spanning class of anything that wants to be able to load a table set,
+    regardless of what it wants to load it from. To do this, it must support a load method
+    that takes a filename and returns the file content in the form:
+        {
+            "Sheet1": [
+                          {...representation of row1 as some kind of dict...},
+                          {...representation of row2 as some kind of dict...}
+                      ],
+            "Sheet2": [...],
+            ...,
+        }
+    Note that at this level of abstraction, we take no position on what form of representation is used
+    for the rows, as long as it is JSON data of some kind. It might be
+         {"col1": "val1", "col2", "val2", ...}
```

Comment:
I think this is probably meant to be:
```suggestion
         {"col1": "val1", "col2": "val2", ...}
```